db/schema.rb

```diff
@@ -17,6 +17,7 @@
     t.integer  "user_id"
     t.integer  "question_id"
     t.string   "content"
+    t.integer  "votes"
```

Comment:
There's a votes table that handles the votes so we can easily manage the votes, I doubt we'd be needing this. same for `comments and questions`